internal/backend/client.go

```diff
@@ -1,9 +1,19 @@
 package backend
 
 import (
+	"fmt"
 	"github.com/dream11/odin/internal/config"
+	"github.com/dream11/odin/internal/ui"
 	"github.com/dream11/odin/pkg/request"
 	"github.com/dream11/odin/pkg/sse"
+	"time"
+)
+
+var logger ui.Logger
+
+const (
```

Comment:
shouldn't these be allowed to be overridden, in the actions themselves?

Reply:
these constants are backend-service specific and will change for each backend-service we could have different retries but for the same backend-service why would we need different retries?